src/manager.rs

```diff
@@ -13,6 +13,8 @@ use super::{
     Errno, Error, EventOps, Events, MutEventSubscriber, Result, SubscriberId, SubscriberOps,
 };
 
+const DEFAULT_READY_EVENTS_CAPACITY: usize = 256;
```

Comment:
It is better to define constants in the scope where they're used. The `DEFAULT_READY_EVENTS_CAPACITY` only makes sense in the context of the `EventManager` implementation. So we can define it as follows:

```rust
impl<S: MutEventSubscriber> EventManager<S> {
    const DEFAULT_READY_EVENTS_CAPACITY: usize = 256;
    /// Create a new `EventManger` object.
    pub fn new() -> Result<Self> {
        Self::new_with_capacity(Self::DEFAULT_READY_EVENTS_CAPACITY)
    }
```

Reply:
done 👍